cas-server-support-pac4j/src/main/resources/META-INF/cas-servlet-pac4j.xml

```diff
@@ -20,25 +18,8 @@
     under the License.
 
 -->
+<beans xmlns="http://www.springframework.org/schema/beans"
```

Comment:
An empty context: kept for consistency with other modules using automatic configuration.